cloudify/utils.py

```diff
@@ -560,6 +560,12 @@ def _get_package_version(plugins_dir, package_name):
             return
         return newest
 
+    @staticmethod
```

Comment:
i hate `@staticmethod` but eh, i'll just do it here for consistency with all the other methods